scss/_buttons.scss

```diff
@@ -70,12 +72,14 @@
 .btn-primary {
   color: #fff;
   text-shadow: 0 -1px 0 rgba(0,0,0,0.15);
-  @include gradient(#8add6d, #60b044);
+  background-color: #60b044;
+  background-image: linear-gradient(#8add6d, #60b044);
```

Comment:
Color literals like `#8add6d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#60b044` should only be used in variable declarations; they should be referred to via variable everywhere else.